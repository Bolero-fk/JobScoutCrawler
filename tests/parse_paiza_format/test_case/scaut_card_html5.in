<div class="MessageCardHeader"><img class="ScoutImage" src="sample5.jpg" alt="スカウト一覧画像"></div><div class="MessageCardBody"><div class="MessageCardTitle"><div class="MessageCardTitleName">LMNエンタープライズ株式会社</div><div class="MessageUnReadPoint" style="display: none;"></div></div><div class="MessageCardText">テスト説明5テスト説明5テスト説明5テスト説明5</div><ul class="MessageCardStatus"><li class="MessageCardStatusItem"><i class="fas fa-yen-sign"></i>  150万 〜 400万円</li><li class="MessageCardStatusItem"><i class="fas fa-map-marker-alt"></i>  愛知県</li><li class="MessageCardStatusItemSub"><i class="fas fa-code"></i> PHP html css</li></ul><div class="MessageCardDate"><span style="margin-right: 0.25rem;">2023/05/13</span><span style="color: rgb(46, 173, 67);">まもなく削除されます</span></div></div>