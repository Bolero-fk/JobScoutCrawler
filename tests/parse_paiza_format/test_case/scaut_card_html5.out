{"company_name": "LMNエンタープライズ株式会社", "min_salary": 150, "max_salary": 400, "location": "愛知県", "using_lang": "PHP html css", "description": "テスト説明5テスト説明5テスト説明5テスト説明5", "recieve_date": "2023/05/13", "remaining_days": 1, "site_name": "paiza"}